### DISCUSSION TITLE
calculating the gas mass?

### DISCUSSION THREAD

**Discussion by KGadallah** [mboquien/cigale#16] [General]
How does X-CIGALE calculate the gas mass? 
I think X-CIGALE employs an assumption to estimate the gas mass because it needs to be derived from sub-mm/radio observation such as CO lines.

**Comment by Guang91** [comment]
The gas mass in CIGALE is from the stellar model, and it's the total gas mass returned to the ISM by evolved stars. So, it's not a direct measurement of the gas content in a galaxy.
